Plate_2_data/2.cellprofiler_ic_processing/loaddata_edit_and_ic.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Please change the /home/jenna part of the path to reflect your machine.
</pre><p> </p><p>I think that the last sentence is no longer needed since you are using relative pathing to instantiate the <code>pathlib.Path</code> object and then executing the <code>pathlib.Path.absolut()</code> method to extract the absolute path. Therefore, this will not require users to change the local root directory. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Ahh great catch! I missed that, you are correct. It is removed.